[cayetano.delgado]improvements to documentation (#16858)
DataDog/documentation: content/en/observability_pipelines/installation.md

> Before installing, make sure you have:

Reviewer: ```suggestion
Before installing, make sure you:
```

> 1. One of the supported Linux Archs: x86_64, AMD64

Reviewer: ```suggestion
1. Are using one of the supported Linux Archs: x86_64 or AMD64
```

> 2. A valid [Datadog API key][5].

Reviewer: ```suggestion
2. Have a valid [Datadog API key][5].
```

> 3. An Observability Pipelines Configuration ID.

Reviewer: ```suggestion
3. Have an Observability Pipelines Configuration ID.
```

    $ sudo apt-get install datadog-observability-pipelines-worker datadog-signing-keys
    ```

> 4. Create a SystemD service file using the [vector.service][7] example as a reference:

Reviewer: ```suggestion
4. Create a SystemD service file using this command:
```

Reviewer: I removed the link to the Vector docs because we're trying to separate out Vector and the Observability Pipelines Worker. Is there a reference we can use for the OPW specifically? If so, we can add that instead.

[4]: https://linux.org/
[5]: /account_management/api-app-keys/#api-keys
[6]: /observability_pipelines/working_with_data/

> [7]: https://github.com/vectordotdev/vector/blob/master/distribution/systemd/vector.service

Reviewer: ```suggestion
```
